sdk/monitor/Azure.Monitor.Query/api/Azure.Monitor.Query.netstandard2.0.cs

```diff
@@ -58,6 +78,25 @@ public enum ServiceVersion
             V2023_05_01_PREVIEW = 1,
         }
     }
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct MetricsQueryAudience : System.IEquatable<Azure.Monitor.Query.MetricsQueryAudience>
+    {
+        private readonly object _dummy;
+        private readonly int _dummyPrimitive;
+        public MetricsQueryAudience(string value) { throw null; }
+        public static Azure.Monitor.Query.MetricsQueryAudience AzureChina { get { throw null; } }
+        public static Azure.Monitor.Query.MetricsQueryAudience AzurePublicCloud { get { throw null; } }
+        public static Azure.Monitor.Query.MetricsQueryAudience AzureUSGovernment { get { throw null; } }
```

Comment:
Seems like the property names should at least match - https://learn.microsoft.com/en-us/dotnet/api/azure.resourcemanager.armenvironment.azuregovernment?view=azure-dotnet#azure-resourcemanager-armenvironment-azuregovernment